package.json

```diff
@@ -110,7 +110,7 @@
   },
   "dependencies": {
     "@digitalbazaar/vc-status-list": "^4.0.0",
-    "@docknetwork/crypto-wasm-ts": "^0.33.3",
+    "@docknetwork/crypto-wasm-ts": "^0.38.1",
```

Comment:
version in package.json of `crypto-wasm-ts` is 0.38.0 on github but 0.38.1 here and on npm? Do you have local changes in `crypto-wasm-ts` that you haven't pushed to github?

Reply:
@lovesh there are no actual changes, but since NPM deploys are manual i had to do it twice (forgot to rebuild first time, just ran npm publish). we can bump the version in crypto-wasm-ts to accomodate